runtime/vm/ContinuationHelpers.cpp

```diff
@@ -1240,7 +1240,7 @@ takeVirtualThreadListToUnblock(J9VMThread *currentThread)
 					/* WAIT/TIMED_WAIT can only be added to blocked list if they have been notified. */
 					Assert_VM_true(J9VMJAVALANGVIRTUALTHREAD_NOTIFIED(currentThread, current->vthread));
 					/* The transition to BLOCKED may have been missed if vthread was in a WAITING state while notified. */
-					if (MM_ObjectAccessBarrierAPI(currentThread).inlineMixedObjectCompareAndSwapU32(
+					if (barrier.inlineMixedObjectCompareAndSwapU32(
 									currentThread,
 									current->vthread,
 									J9VMJAVALANGVIRTUALTHREAD_STATE_OFFSET(currentThread),
```

Comment:
Please correct the indentation here - two tabs more than line 1243 is sufficient.